### PR TITLE
feat(vouchers): add v1 that extends v0

### PR DESCRIPTION
**Pull request by jmy-kellogg**

### CODE REVIEW

File: src/v1/vouchers.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import { Client } from '../client'
+import { BaseError } from '../errors'
+import { UriHelper } from '../uri-helper'
+import * as v0 from '../v0'
```
Comment (Endebert) [inline src/v1/vouchers.ts]: This import is way too big for what is necessary here, but probably doesn't matter as it's cached.

Still, why not only: `import { Vouchers as VouchersV0, VouchersOptions } from from '../v0/vouchers'`

Comment (jmy-kellogg) [reply src/v1/vouchers.ts]: That's definitely a good point. removed import all

Comment (Endebert) [reply src/v1/vouchers.ts]: :ok_hand:

**Pull request merged**